### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by shawe**
- Fixes to pass Travis test.
- Little change to correct execution of Travis for any user, not only for NeoRazorX Travis account.
- Replaced input-group-btn with input-group-prepend (https://getbootstrap.com/docs/4.0/migration/#input-groups)
- Removed unneeded ignored files on this branch.

### CODE REVIEW

**Comment by NeoRazorX** [comment]
Please, resolve the conflicts.

**Comment by shawe** [comment]
Ready.

File: .gitignore [hunk 1]
```diff
@@ -1,10 +1,3 @@
-###> symfony/framework-bundle ###
```
Comment (shakaran) [inline .gitignore]: Not sure if this should be deleted. Also it should be made in another issue or ticket. A good practice is make atomic changes over a subject or issue, then make a PR only for that, then checkout master and make another branch with other different changes or issue with other PR. Good practice to read: https://seesparkbox.com/foundry/atomic_commits_with_git
Probably this time is ok go for merge, but in future we need to organize smart for avoid conflicts ;)

File: Core/App/AppInstaller.php [hunk 2]
```diff
@@ -70,11 +70,14 @@ public function __construct()
 
         if ($installed) {
             header('Location: ' . $this->getUri());
-        } else {
-            $this->render();
         }
-    }
 
+        if ($this->request->getBasePath() !== $this->getUri()) {
+            header('Location: ' . $this->request->getBasePath());
```
Comment (shakaran) [inline Core/App/AppInstaller.php]: Maybe is more appropiate use the symfony way or the framework with a [RedirectReponse](https://api.symfony.com/3.4/Symfony/Component/HttpFoundation/RedirectResponse.html) object

File: Core/Base/PluginManager.php [hunk 3]
```diff
@@ -305,6 +305,7 @@ private function getPluginInfo($pluginName, $iniContent)
 
             if ($info['min_version'] >= 2018 && $info['min_version'] <= self::MIN_VERSION) {
                 $info['compatible'] = true;
+                $info['description'] = self::$i18n->trans('compatible');
```
Comment (shakaran) [inline Core/Base/PluginManager.php]: Small rule of thumb, always that you add a new translation key, it should be added at least in spanish and english locale files, it will make more ease to keep the l18n files updated

File: Core/Lib/ExtendedController/WidgetItemAutocomplete.php [hunk 4]
```diff
@@ -117,7 +117,7 @@ public function getEditHTML($value)
             $html .= '<span class="input-group-prepend"><span class="input-group-text">'
                 . '<i class="fa fa-keyboard-o" aria-hidden="true"></i></span></span>';
         } else {
-            $html .= '<span class="input-group-btn">'
+            $html .= '<span class="input-group-prepend">'
```
Comment (shakaran) [inline Core/Lib/ExtendedController/WidgetItemAutocomplete.php]: This css changes, probably should go in another different PR, but is OK go to merge, but try next time split per issue ;)

File: .gitignore (hunk 1 above)
Comment (shawe) [reply .gitignore]: Yes, this a bad practice from my part that I do a lot, feel free to say me again if this happens another time because I need to correct that.

This link also may be a "must read" in our manual for new git contributors.

File: Core/App/AppInstaller.php (hunk 2 above)
Comment (shawe) [reply Core/App/AppInstaller.php]: Yes, but in this case, I only replace uneeded elses.

File: Core/Base/PluginManager.php (hunk 3 above)
Comment (shawe) [reply Core/Base/PluginManager.php]: I know that, I found a lot of missing translation in past that I add after browsing from new feature added. This time is my fault ;)

File: Core/Lib/ExtendedController/WidgetItemAutocomplete.php (hunk 4 above)
Comment (shawe) [reply Core/Lib/ExtendedController/WidgetItemAutocomplete.php]: This was the real begin of this PR.

**Comment by shawe** [comment]
@NeoRazorX I know that is not the correct type of PR, but now is do it this way, if you want that I add also the suggestions from @shakaran, confirm it (including do it again in separate atomic PR).

And don't worry if you need to call my attention again to avoid that I do it this way, I'm aware that I need to correct that.

**Comment by NeoRazorX** [comment]
Yes, make atomic PR.

**Comment by shawe** [comment]
I Close this PR and make a new PR for each type of change.
